### PR TITLE
xinput_calibrator: update to 0.8.0, fix distfile

### PR DESCRIPTION
**Pull request by meator**
related to #52319

#### Testing the changes
- I tested the changes in this PR: **NO**

### CODE REVIEW

File: srcpkgs/xinput_calibrator/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'xinput_calibrator'
 pkgname=xinput_calibrator
-version=0.7.5
-revision=3
+version=0.8.0
+revision=1
 build_style=gnu-configure
```
Comment (classabbyamp) [inline srcpkgs/xinput_calibrator/template:5]: switch to meson?

File: srcpkgs/xinput_calibrator/template [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 hostmakedepends="pkg-config"
 makedepends="libXrandr-devel libXi-devel"
 short_desc="Utility to calibrate X input devices"
 maintainer="Michael Aldridge <maldridge@voidlinux.org>"
 license="X11"
 homepage="http://www.freedesktop.org/wiki/Software/xinput_calibrator"
```
Comment (classabbyamp) [inline srcpkgs/xinput_calibrator/template:11]: https

File: srcpkgs/xinput_calibrator/template [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-distfiles="http://github.com/downloads/tias/xinput_calibrator/$pkgname-$version.tar.gz"
-checksum=baa4ddca49ec94c27ba4c715bfa26692fec1132103e927213c3169e475d3d971
+distfiles="https://xorg.freedesktop.org/archive/individual/app/xinput_calibrator-${version}.tar.xz"
```
Comment (classabbyamp) [inline srcpkgs/xinput_calibrator/template:12]: changelog: https://gitlab.freedesktop.org/xorg/app/xinput-calibrator/-/raw/master/Changelog

File: srcpkgs/xinput_calibrator/template (hunk 1 above)
Comment (meator) [reply srcpkgs/xinput_calibrator/template:5]: Ah, I haven't noticed that [they have added Meson](https://lists.x.org/archives/xorg-announce/2024-July/003522.html).

Comment (meator) [reply srcpkgs/xinput_calibrator/template]: I had to add a patch because of this, but it's working.

**Pull request merged**